template.rb

```diff
@@ -112,14 +112,21 @@ def render_file(filename)
 
   # Prevent test noise in generators
 
-  application \
-    <<-RB.gsub(/^      /, "")
-      config.generators do |g|
+  load_defaults = open("config/application.rb").read.to_s.match(/config.load_defaults \d.\d\n/)
+
+  inside "config" do
+    insert_into_file "application.rb",
+      after: load_defaults[0] do
+      <<-RUBY.gsub(/^      /, "")
+          config.generators do |g|
             g.test_framework nil
             g.factory_bot false
             g.scaffold_stylesheet false
           end
-    RB
+          config.action_controller.allow_forgery_protection = false
```

Comment:
@jelaniwoods I'm still not quite clear on this; don't we want something like this?

https://github.com/raghubetina/test-csrf/blob/2a5bca8d6e498b4fcdf99e4a4158decf68b3f680/config/application.rb#L19

Reply:
@raghubetina That's essentially what this generates. It just adds disables some of the generated code as well. I tried to make it flexible so that it would work with a different Rails version if we needed that before fall. 

![new-config-application](https://user-images.githubusercontent.com/17581658/59882095-e93ca300-9376-11e9-8f46-19c811b1908a.PNG)

Reply:
But don't we want `default_protect_from_forgery` instead of `allow_forgery_protection`?

Reply:
What a blunder! You're correct, I'll make the change.